packages/now-cli/src/commands/inspect.js

```diff
@@ -70,7 +65,7 @@ export default async function main(ctx) {
   const { print, log, error } = output;
 
   // extract the first parameter
-  id = argv._[1];
+  const [deploymentIdOrHost] = argv._;
```

Comment:
Are you sure this is right? It was using index 1 before? So I would have expected something like:

```suggestion
  const [_, deploymentIdOrHost] = argv._;
```